dummy.js

```diff
@@ -0,0 +1,104 @@
+
+'use strict';
+
+// ======= packages =======
+
+// this implies that express has already been downloaded via npm. the command for downloading and saving it is  `npm install -S express`
+
+const express = require('express');
+const cors = require('cors');
+// const { response } = require('express');
+require('dotenv').config();
+
+// =====setup application (server) ========
+
+const app = express(); // creates server from the Express library
+app.use(cors());
+// loads the middleware. We are loadig cors so that requests don't get blocked.
+
+// ===== Global variables =====
+// process.env.PORT references the the PORT env variable from the terminal
+
+const PORT = process.env.PORT || 3005;
+
+// ===== Routes =====
+// app.get: attaches a listener of method type GET to the server with a (route and a callback)
+// '/': route you can visit the server at localhost:3000/ and trigger the callback.
+// (request, response) => : the callback function
+// request: data requeted by client
+// response : data sent back to the client
+// response.send (<anything>): takes the argument and send it back to the client
+
+app.get('/', (req, res) => {
+  res.send(`<h1>This server is running on port ${PORT}</h1>`);
+});
+
+
+// expect a key value pair of name:rasta and lastName:dog
+// send `rasta dog`
+// this lives with all the client data, in the `request (req)` parameter
+// inside request will always live a property  query: { name: 'rasta', lastName: 'dog' },
+
+
+// create a route with an end point of location
+// location request/send
+
+app.get('/location', (req, res) => {
+  console.log(req.query.city);
+  res.send({
+    'search_query': 'seattle',
```

Comment:
Once you pull in the /data/location.json data, you can plug that into your Location constructor with newLocation = new Location( req.query.city, jsonData.formatted_query, jsonData.latitude, jsonData.longitude )

THEN, you can return that object with your res.send() function.